crates/noirc_frontend/src/node_interner.rs

```diff
@@ -138,6 +138,8 @@ impl FuncId {
     }
 }
 
+pub type TestFunc = (FuncId, /* should fail */ bool);
```

Comment:
This is a type alias instead of a new type. You want to define this as a struct with named fields.

Reply:
Can we also avoid abbreviations like `Func` over `Function`?